jni/src/main/java/com/horizen/sc2scnative/Sc2ScProof.java

```diff
@@ -0,0 +1,106 @@
+package com.horizen.sc2scnative;
```

Comment:
I'm waiting @paolocappelletti answer before change it... if also @paolocappelletti like it I'll not change it.

Reply:
Ok, I found an issue here. The only way to build a `Sc2ScProof` is from `Sc2Sc.createProof()`. I can add something like `deserialize` but it seams quite bit synthetic because the implementation cold just copy the bytes ... in addition I can try to call a native method to validate the bytes but it seams a little bit exaggerated. 

So I'll follow the @DanieleDiBenedetto hint and remove this class.

Reply:
I've moved the `verify` stuff in `Sc2Sc` and removed the useless `Sc2ScProof` class as @DanieleDiBenedetto suggested.